content/collective_intro.tex

```diff
@@ -8,11 +8,17 @@
 \FUNC{shmem\_sync\_all} must be called by all \acp{PE} of the \openshmem program.
 
 The active set is defined by the arguments \VAR{PE\_start}, \VAR{logPE\_stride},
-and \VAR{PE\_size}.  \VAR{PE\_start} is the starting \ac{PE} number, a log (base
-2) of \VAR{logPE\_stride} is the stride between \acp{PE}, and \VAR{PE\_size} is
-the number of \acp{PE} participating in the active set.  All \acp{PE}
-participating in the collective routine must provide the same values for these
-arguments.
+and \VAR{PE\_size}.  \VAR{PE\_start} specifies the starting \ac{PE} number and
+is the lowest numbered PE in the active set.  The stride between successive
+\acp{PE} in the active set is $2^{logPE\_stride}$ and \VAR{logPE\_stride} must
+be greater than or equal to zero.  \VAR{PE\_size} specifies the number of
+\acp{PE} in the active set and must be greater than zero.  The active set must
```

Comment:
Follow-up with @swpoole on size greater or equal to zero.

Reply:
@swpoole Thoughts on this?  Do we need to allow `PE_size` equal to zero in the active set?

Reply:
@swpoole This proposal is moving into voting.  You had wanted to check on whether apps need us to allow `PE_size == 0` in collectives.  Any thoughts on this?  Currently, the behavior is undefined when a PE not in the active set calls a collective routine.  This implies that behavior is also undefined when `PE_size` is `0` since no PEs are in the active set.  Thus, the changes proposed in here are not restricting semantics.